### ISSUE TITLE
request

### ISSUE THREAD

**Issue by noobyoume** [UnamSanctam/SilentCryptoMiner#2261]
can you add flux coin (zelhash) in your miner. as this is the only coin which requires min 2 gb vram. as per my knowledge.

**Comment by UnamSanctam** [comment]
Since it's a completely different family compared to the current algorithms then it will take a lot of work, though I do plan to add more algorithms if I have the time, but there are many kawpow coins that have a lower than 2GB VRAM requirement, such as Clore or Neurai.

**Comment by noobyoume** [comment]
so i can mine clore coin if i selected kawpow algorithm in builder ??  which coin will you suggest me to mine on 2 gb graphic card ?

**Comment by UnamSanctam** [comment]
Yes, and Neurai requires even less at only 1.48GB of VRAM at the moment: https://minerstat.com/dag-size-calculator.

**Comment by noobyoume** [comment]
also while running gpu miner. pc always get crashed dont know the reason ? pc have 2 gb graphic card. and neurai mining..

**Comment by UnamSanctam** [comment]
Possibly something with your GPU, like an old driver or something wrong. And you don't have the BSoD setting enabled in the miner?
